griptape/tools/file_manager/tool.py

```diff
@@ -30,13 +31,18 @@ def load_files_from_disk(self, params: dict) -> ListArtifact | ErrorArtifact:
 
             try:
                 with open(full_path, "rb") as file:
-                    list_artifact.value.append(
-                        BlobArtifact(
+                    if self.encoding is not None:
```

Comment:
Can just be `if self.encoding:`

Reply:
We should also make it more descriptive. Something like `load_file_encoding`?

Wondering if we should integrate it into `_save_to_disk` as well.... Then the question becomes if we use the same `encoding` field for both or if we want to have `load_file_encoding` and `save_file_encoding`.